### ISSUE TITLE
Adding clients to whisperlist doesn't work anymore

### ISSUE THREAD

**Issue by Bluscream** [Splamy/TS3AudioBot#142]
Since the last commits subscribing to a single client doesn't work anymore for some reason :/

```
<12:05:12> "Bluscream": !help subscribe
<12:05:13> "I ♥ Radio | !sub | !unsub": The command contains the following subfunctions: , tempchannel, channel
<12:05:34> "Bluscream": !sub
<12:05:34> "I ♥ Radio | !sub | !unsub": Error: Ambiguous command, possible names: , tempchannel, channel, ...
<12:05:36> "Bluscream": !sub me
```

**Comment by Splamy** [comment]
Aw damn, I knew I missed something :P

**Comment by Bluscream** [comment]
Fixed with 83c27fc47002857862ced49085a8772c6ebb6c9e
